<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>5.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="editSearch">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonFind">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="autoFormatting">
       <set>QTextEdit::AutoNone</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editId"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фамилия Имя: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Год Рождение: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editBrth"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Пол: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findEmploye()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addEmploye()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveEmploye()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findEmploye()</slot>
  <slot>addEmploye()</slot>
  <slot>saveEmploye()</slot>
 </slots>
</ui>
